services-toolkit/how-to-guides/configure-dynamic-provisioning-tanzu-postgresql.hbs.md

```diff
@@ -0,0 +1,354 @@
+# Configure Dynamic Provisioning of VMware Tanzu Postgres Service Instances
+
+## <a id="stk-dynamic-provisioning-tanzu-psql-about"></a> About
+
+This document details the steps required to setup dynamic provisioning of VMware Tanzu Postgres service instances. If you are not already familiar with dynamic provisioning in Tanzu Application Platform, it is recommended to first run through the tutorial [Setup Dynamic Provisioning of Service Instances](../tutorials/setup-dynamic-provisioning.hbs.md). The learnings from that tutorial will be beneficial to understanding the steps presented here.
```

Comment:
**thought/non-blocking** Would be good to an equivalent of MySQL and Rabbitmq here too. This can come after though.

Reply:
Yeah I agree that would be good. Those how-to guides should be pretty similar to this one. I'll add them if I have time.